red_queen/cli.py

```diff
@@ -0,0 +1,185 @@
+# ------------------------------------------------------------------------------
+# Part of Red Queen Project.  This file is distributed under the Apache 2.0.
+# See accompanying file /LICENSE for details.
+# ------------------------------------------------------------------------------
+"""Shebang is used to make this code an python executable"""
+#!/usr/bin/env python3
```

Comment:
This should be the first line in the file.

Reply:
Which line should be the first line?

Reply:
The shebang should be line number 1 if we want your Unix shell to interpret it